### ISSUE TITLE
Empty (black) status bar when switching theme.

### ISSUE THREAD

**Issue by Rom1-GoPlay** [mslalith/focus_launcher#305]
Hello !
When I turn the dark theme on or off, the status bar turns black.

![Screenshot_20231019_153040_Focus Launcher.jpg](https://github.com/mslalith/focus_launcher/assets/131159819/8b84503b-bc00-4fcc-b6e0-6197e3b327f4)

To make it displaying again, I need to open and close the notification center.

**Comment by mslalith** [comment]
Do you see this issue always? I'm not able to reproduce. Can you provide your device details and a recording would also help

**Comment by Rom1-GoPlay** [comment]
Yes, the status bar turns black each time I change the dark mode (on / off).

My device is a Samsung Galaxy A51 running Android 13 (and One UI 5.1).

I just realised that the issue is also visible when: focus launcher is not the default launcher and I open it.

I can also clarify somthing: As I said, to make the status bar display again, I need to open and close the notification center but I need to open it from the status bar (when I open the notification center by pulling down, the status bar is still black when I close).

**Comment by mslalith** [comment]
Some changes were made to status and navigation bar in [v0.8.0](https://github.com/mslalith/focus_launcher/releases/tag/v0.8.0). Can you try with that and lemme know if this still persists.

**Comment by Rom1-GoPlay** [comment]
The problem is still the same in 0.8.0

**Comment by mslalith** [comment]
I don't see the issue. Can you share a recording?

**Comment by Rom1-GoPlay** [comment]
yes

https://github.com/mslalith/focus_launcher/assets/131159819/9cfc283a-1539-43c1-af13-2f1d6d335de8
